R/install.R

```diff
@@ -31,11 +31,11 @@
 #' url <- "https://cran.r-project.org/bin/windows/base/R-2.15.3-win.exe"
 #' file.name.from.url(url) # returns: "R-2.15.3-win.exe"
 #' }
-file.name.from.url <- function(URL) {
- #  tail(strsplit(URL,   "/")[[1]],1)
-   # corrected to use R's base function thanks to Uwe's remark.
-   basename(URL)
-}
+file.name.from.url <- function(URL, rm.params=FALSE) {
```

Comment:
Please also edit the roxygen2 comment above by adding text about the parameter you've added.
(adding something in the examples would also be nice)
Once done, you'll need to recompile the code to recreate the help file. Otherwise, this wouldn't pass CRAN checks.

Thanks.

Reply:
Sorry, I forgot about this. Will try and sort it now.

Reply:
@talgalili done. It should pass CRAN's requirements now.